cmd/services.rb

```diff
@@ -33,6 +33,7 @@ def services_args
         [`sudo`] `brew services cleanup`:
         Remove all unused services.
       EOS
+      flag "--file=", description: "Use the plist file from this location. Use `--file=-` to pipe to stdin/stdout."
```

Comment:
This is file but I'd like to see the existing syntax supported (if it's not already?) and `odeprecated` for a while and then `odisabled`

Reply:
It isn't, I'll check for that.